contracts/core.clar

```diff
@@ -0,0 +1,180 @@
+(define-constant CONTRACT_OWNER tx-sender)
+
+(define-constant ERR_UNAUTHORIZED u1000)
+(define-constant ERR_CONTRACT_ALREADY_EXISTS u1001)
+(define-constant ERR_CONTRACT_DO_NOT_EXISTS u1002)
```

Comment:
Slight change in the grammar here, suggested for all other spots the constant is used in the contract and tests as well.

```suggestion
(define-constant ERR_CONTRACT_DOES_NOT_EXIST u1002)
```